modules/standard/Random.chpl

```diff
@@ -1068,6 +1052,27 @@ module Random {
     }
 
 
+    /*
+
+     Low-level PCG RNG implementation.
+
+     Besides a Chapel :class:`RandomStream` interface, the :mod:`PCGRandomLib`
```

Comment:
w.r.t. this introductory clause, I may not be tracking properly, but this sub-submodule doesn't still supply a RandomStream interface does it?

In retrospect, I wonder if there's a better name for this module.  Namely, if you asked me out of the blue "What's the difference between PCGRandom and PCGRandomLib, I'm not sure I have any intuition.  The latter is what Kyle was considering the "system" interface, but I'm not wild about that label/distinction either.  PCGRandomLowLevel is the first thing that comes to mind for me, though that's somewhat of a mouthful...  or PCGRandomUtil?  PCGRandomHelp?